saq/queue/postgres.py

```diff
@@ -61,11 +63,7 @@ class PostgresQueue(Queue):
         stats_table: name of the Postgres table SAQ will write stats to (default "saq_stats")
         dump: lambda that takes a dictionary and outputs bytes (default `json.dumps`)
         load: lambda that takes str or bytes and outputs a python dictionary (default `json.loads`)
-        min_size: minimum pool size. (default 4)
-            The minimum number of Postgres connections.
-        max_size: maximum pool size. (default 20)
-            If greater than 0, this limits the maximum number of connections to Postgres.
-            Otherwise, maintain `min_size` number of connections.
```

Comment:
Moved to the `from_url` method since it can't be modified after initial creation.